ROMFS/px4fmu_common/init.d/airframes/13013_deltaquad

```diff
@@ -31,6 +31,8 @@ param set-default BAT1_R_INTERNAL 0.0025
 param set-default CBRK_AIRSPD_CHK 162128
 param set-default CBRK_IO_SAFETY 22027
 
+param set-default COM_FW_PERM_STAB 1
```

Comment:
@sfuhrer Need to check but if it's now possible to have stabilized mode mapped for multirotors then I suggest we get rid of this completely. In the past this was added because there was not stabilized mode for MC, just manual. So there was not way to fly manual MC, transition and end up in stabilized FW.

Reply:
Very happy to remove it completely yes! I think in general it's a very bad idea to fly "Manual" in MC for VTOL, Stabilized it much clearer in that case. And if the user is flying in Manual FW and transitions to MC, then it's anyway clear that the system automatically enables stabilization (=implicit use of Stabilized mode, hopefully we can clean that up as well soonish).

Reply:
I removed it.